WebCLI/forms.py

```diff
@@ -100,9 +105,11 @@ class Meta:
         widgets = {
             'name': TextInput(),
             'basis_set': Select(choices=((x, x) for x in basis_set_options())),
-            'transformation': TextInput(),
-            'structure': Textarea(attrs={'rows': 6, 'cols': 50}),
-            'active_orbitals': Textarea(attrs={'rows': 6, 'cols': 50}),
+            'transformation': TextInput(attrs={'placeholder': 'Can be left empty'}),
+            'structure': Textarea(attrs={'rows': 6, 'cols': 50,
+                                         'placeholder': 'e.g. Li 0.0 0.0 1.6'}),
+            'active_orbitals': Textarea(attrs={'rows': 6, 'cols': 50,
+                                               'placeholder': 'For example "A1 1"'}),
         }
```

Comment:
Slightly nitpicky but I'd prefer the same format for all placeholders ("e.g. \<example input>").

Reply:
Corrected'd